pumpkin/src/lan_broadcast.rs

```diff
@@ -0,0 +1,39 @@
+use pumpkin_config::ADVANCED_CONFIG;
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+use std::time::Duration;
+use tokio::net::UdpSocket;
+use tokio::time;
+
+const BROADCAST_ADDRESS: SocketAddr =
+    SocketAddr::new(IpAddr::V4(Ipv4Addr::new(224, 0, 2, 60)), 4445);
+
+pub async fn start_lan_broadcast(bound_addr: SocketAddr) {
+    let port = ADVANCED_CONFIG.lan_broadcast.port.unwrap_or(0);
+
+    let socket = UdpSocket::bind(SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), port))
```

Comment:
```suggestion
    let socket = UdpSocket::bind(format!("0.0.0.0:{port}"))
```
This looks cleaner and easier to read.

Reply:
Isn't SocketAddr::new more performant ?

Reply:
No because the compiler should optimize this and make it performant (it should compile to the same assembly at the end).